src/main/resources/graphql/flashcards.graphqls

```diff
@@ -0,0 +1,32 @@
+"""
+A flashcard is a set of two or more sides. Each side has a label and a text.
+The label is used to specify which side of the flashcard is being shown to the user first for learning
+and which sides he has to guess.
+"""
+type Flashcard {
+    """
+    Unique identifier of this flashcard.
+    """
+    id: UUID!
+    """
+    List of sides of this flashcard.
+    """
+    sides: [FlashcardSide!]!
```

Comment:
Is the FlashcardSide type defined somewhere already? I can't find it.

Reply:
Yes it is in this file below

Reply:
For some reason this changes are not in the IntelliJ commit dialog but only visible in the command line... wtf?